ansible_collections/arista/avd/requirements.txt

```diff
@@ -10,4 +10,4 @@ deepmerge>=1.1.0
 cryptography>=38.0.4
 # No anta requirement until the eos_validate_state integration is out of preview.
 # anta>=1.0.0
-betterproto==2.0.0b6
+aristaproto>0.1.1
```

Comment:
it says `>` so I assume there is a new version to be released on pypi that will be bigger - for now only `0.1.0` is out?

Reply:
Fixed